.github/workflows/linux.yml

```diff
@@ -223,10 +220,11 @@ jobs:
       - name: Download OpenVINO package
         uses: actions/download-artifact@v4
         with:
-          name: openvino_package
+          name: ${{ env.OV_ARTIFACT_NAME }}
           path: ${{ env.INSTALL_DIR }}
 
       - name: Extract OpenVINO packages
+        if: needs.openvino_build.result == 'success'
```

Comment:
@mryzhov maybe we can drop build stage? and always rely on provider

Reply:
if we remove an ability to manually build OV from sources then we can drop this step. 
Another solution - do not extract artifacts automatically

Reply:
Dropped build stage